module-apps/application-messages/include/application-messages/Constants.hpp

```diff
@@ -18,6 +18,7 @@ namespace gui::name::window
     inline constexpr auto thread_options    = "ThreadOptions";
     inline constexpr auto sms_options       = "SMSOptions";
     inline constexpr auto sms_templates     = "SMSTemplates";
+    inline constexpr auto call_sms_templates = "CallSMSTemplates";
```

Comment:
It was a good thing to eventually rename it so that it's clear that this thing is about calls :)

Reply:
tried my best :P